frontend/src/metabase/ui/components/inputs/Checkbox/Checkbox.stories.mdx

```diff
@@ -52,96 +52,105 @@ Checkbox buttons allow users to select a single option from a list of mutually e
 
 ## Examples
 
-export const Template = args => (
+export const DefaultTemplate = args => <Checkbox {...args} />;
+
+export const CheckboxGroupTemplate = args => (
```

Comment:
Here args are not used and we probably don't want to use them (controls are for the checkbox but here we're showcasing how to group them in a checkbox group).
Do we want to remove the controls or are we fine with some stories having controls not working if it makes the code easier to manage and if it's clear that they shouldn't work?
I think if we wanted to disabled them we'd have to set each control to disabled which is kinda tedious so I'd be fine like this but just wanted to make sure

Reply:
I've passed the args. I think it makes sense to have them, e.g. to test different `labelPosition` in a CheckboxGroup.

Reply:
Also about

> Do we want to remove the controls or are we fine with some stories having controls not working if it makes the code easier to manage and if it's clear that they shouldn't work?

I've been thinking about this and I'm fine with this for now. We could exclude some `argTypes` later on if it causes trouble.